service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/AdditionalApplicationsUploadedEventHandlerTest.java

```diff
@@ -0,0 +1,166 @@
+
+        given(hmctsCourtLookupConfiguration.getCourt(caseData.getCaseLocalAuthority()))
+            .willReturn(new HmctsCourtLookupConfiguration.Court(COURT_NAME, "hmcts-non-admin@test.com",
+                COURT_CODE));
+
+        given(additionalApplicationsUploadedEmailContentProvider.getNotifyData(caseData))
+            .willReturn(additionalApplicationsParameters);
+
+        additionalApplicationsUploadedEventHandler.notifyAdmin(new AdditionalApplicationsUploadedEvent(caseData));
+
+        verify(notificationService).sendEmail(
+            INTERLOCUTORY_UPLOAD_NOTIFICATION_TEMPLATE,
+            "hmcts-non-admin@test.com",
+            additionalApplicationsParameters,
+            caseData.getId());
+    }
+
+    @Test
+    void shouldNotifyCtscAdminOnAdditionalApplicationsUploadWhenCtscIsEnabled() {
+        CaseData caseData = CaseData.builder()
+            .id(RandomUtils.nextLong())
+            .sendToCtsc("Yes")
+            .build();
+
+        given(idamClient.getUserInfo(AUTH_TOKEN)).willReturn(
+            UserInfo.builder().sub(CTSC_INBOX).roles(LOCAL_AUTHORITY.getRoleNames()).build());
+
+        given(inboxLookupService.getRecipients(
+            LocalAuthorityInboxRecipientsRequest.builder().caseData(caseData).build()))
+            .willReturn(Set.of(LOCAL_AUTHORITY_EMAIL_ADDRESS));
+
+        given(additionalApplicationsUploadedEmailContentProvider
+            .getNotifyData(caseData))
+            .willReturn(additionalApplicationsParameters);
+
+        additionalApplicationsUploadedEventHandler.notifyAdmin(new AdditionalApplicationsUploadedEvent(caseData));
+
+        verify(notificationService).sendEmail(
+            INTERLOCUTORY_UPLOAD_NOTIFICATION_TEMPLATE,
+            CTSC_INBOX,
+            additionalApplicationsParameters,
+            caseData.getId());
+    }
+
+    @Test
+    void shouldNotNotifyHmctsAdminOnAdditionalApplicationsUpload() {
+        CaseData caseData = caseData();
+
+        given(requestData.userRoles()).willReturn(new HashSet<>(Arrays.asList("caseworker", "caseworker-publiclaw",
+            "caseworker-publiclaw-courtadmin")));
+
+        additionalApplicationsUploadedEventHandler.notifyAdmin(new AdditionalApplicationsUploadedEvent(caseData));
+
+        verifyNoInteractions(notificationService);
+    }
+
+    private AdditionalApplicationsUploadedTemplate getAdditionalApplicationsUploadedTemplateParameters() {
+        String fileContent = new String(Base64.encodeBase64(DOCUMENT_CONTENT), ISO_8859_1);
+        JSONObject jsonFileObject = new JSONObject().put("file", fileContent);
+
+        return AdditionalApplicationsUploadedTemplate.builder()
+            .callout(subjectLine)
+            .respondentLastName("Smith")
+            .caseUrl("null/case/" + JURISDICTION + "/" + CASE_TYPE + "/12345#C2Tab")
+            .documentLink(jsonFileObject.toMap())
+            .applicationTypes(Arrays.asList("C2", "C13A - Special guardianship order"))
+            .build();
```

Comment:
might be worth adding a handler test for AdditionalApplicationsPbaPaymentNotTakenEventHandler